sailfishusermanager.cpp

```diff
@@ -196,7 +198,7 @@ bool SailfishUserManager::copyDir(const QString &source, const QString &destinat
     return true;
 }
 
-bool SailfishUserManager::makeHome(const QString &user)
+bool SailfishUserManager::makeHome(const QString &user, bool suwDone)
```

Comment:
Am I missing something as  I don't see makeHome() being called so that suwDone is true.